abdulfataiaka/react-nyne
# #160785807 Write test suites for layout components

#### What does this PR do?
- Adds test suites for game interface layout components 

#### Description of Task to be completed?
- Configure jest
- Write tests cases for the different layout components implemented

#### How should this be manually tested?
- Clone the repository
- Checkout int the branch `ft-test-interface-components-160785807`
- Install all dependencies
- Run `yarn run tests` or `jest (if install globally)` to ensure all test cases are passing

#### What are the relevant pivotal tracker stories?
- [#160785807](https://www.pivotaltracker.com/story/show/160785807)

#### Screenshots (If Applicable)
<img width="586" alt="screen shot 2018-09-27 at 2 01 34 am" src="https://user-images.githubusercontent.com/32123313/46117472-4ac14e00-c1f9-11e8-9ca4-9cad3c3ae785.png">
